src/platform/packages/shared/kbn-unified-tabs/src/components/tab/tab.tsx

```diff
@@ -257,10 +257,9 @@ export const Tab: React.FC<TabProps> = (props) => {
             )}
             {!!onClose && (
               <EuiFlexItem grow={false} className="unifiedTabs__closeTabBtn">
-                <EuiToolTip content={closeButtonLabel}>
+                <EuiToolTip content={closeButtonLabel} disableScreenReaderOutput>
                   <EuiButtonIcon
-                    // semantically role="tablist" does not allow other buttons in tabs
-                    aria-hidden={true}
+                    aria-label={closeButtonLabel}
```

Comment:
Can we keep `aria-hidden={true}` here?

Reply:
Could you please explain in more details your case here? Ask just because of 
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-hidden

and warning  message here 

> Warning: Do not use aria-hidden="true" on focusable elements.

Reply:
Otherwise it creates a11y warnings that `role="tablist"` does not allow other buttons in tabs. Open to other ideas on how to solve this conflict in the specifications.